test/cloudops.go

```diff
@@ -59,6 +64,36 @@ func compute(t *testing.T, driver cloudops.Ops) {
 	groupInfo, err := driver.InspectInstanceGroupForInstance(instanceID)
 	require.NoError(t, err, "failed to inspect instance group")
 	require.NotNil(t, groupInfo, "got nil instance group info from inspect")
+
+	err = driver.SetInstanceGroupSize(groupInfo.Name, groupInfo.Zone, clusterNodeCount, 5*time.Minute)
+	if err != nil {
+		_, ok := err.(*cloudops.ErrNotSupported)
+		if !ok {
+			t.Errorf("failed to set node count. Error:[%v]", err)
+		}
+	}
+
+	currentCount, err := driver.GetClusterSizeForInstance(instanceID)
+	if err != nil {
+		_, ok := err.(*cloudops.ErrNotSupported)
+		if !ok {
+			t.Errorf("failed to get node count. Error:[%v]", err)
+		}
+	} else {
+		// clusterNodeCount is per availability zone.
+		// So total cluster-wide node count is clusterNodeCount*num. of az
+		require.Equal(t, int64(clusterNodeCount*len(groupInfo.Zones)), currentCount,
+			"expected cluster node count does not match with actual node count")
+	}
+
+	// Validate when timeout is given as 0, API does not error out.
+	err = driver.SetInstanceGroupSize(groupInfo.Name, groupInfo.Zone, clusterNodeCount+1, 0)
+	if err != nil {
+		_, ok := err.(*cloudops.ErrNotSupported)
+		if !ok {
+			t.Errorf("Fialed to set node count. Error:[%v]", err)
+		}
+	}
```

Comment:
the UT doesn't check if the instance group size actually got updated. You need to add a GetClusterSize.. call after that.